app/models/user.rb

```diff
@@ -479,6 +480,12 @@ def send_email_to_all_addresses(user_mailer_template)
     end
   end
 
+  def reload(...)
+    remove_instance_variable(:@active_profile) if defined?(@active_profile)
```

Comment:
is this still needed with the above changes?

Reply:
Hmm, i guess remove_instance_variable is only needed for pending, let me update